### PR TITLE
Last seen at

### PR DESCRIPTION
**Pull request by ikerreyes**
- remove `last_checked_at` when empty for /clusters endpoint to match actual behavior
- remove duplicated sorting related tests in ClustersListTable

### CODE REVIEW

**Review by gkarat — APPROVED** [review]
appears to be clean, see no problems here 👍🏼

File: src/Components/ClustersListTable/ClustersListTable.spec.ct.js [hunk 1]
```diff
@@ -106,7 +99,7 @@ describe('data', () => {
       .should('be.gte', 1);
   });
   it('at least one entry does not have last seen', () => {
-    cy.wrap(_.filter(data, (it) => it.last_checked_at === ''))
+    cy.wrap(_.filter(data, (it) => it.last_checked_at === undefined))
```
[the remark this answers is not in the document]
Comment (ikerreyes) [reply src/Components/ClustersListTable/ClustersListTable.spec.ct.js:102]: yes, but I want to ensure that the data has that field missing. Doing `!it.last_checked_at` will also be fine if the field is 0, or empty string or even `false`.

[the remark this answers is not in the document]
Comment (gkarat) [reply src/Components/ClustersListTable/ClustersListTable.spec.ct.js:102]: ah, ok, then it makes sense

File: src/Components/AffectedClustersTable/AffectedClustersTable.spec.ct.js [hunk 2]
```diff
@@ -78,6 +78,7 @@ describe('test data', () => {
       expect(arr).to.include('not existing cluster');
     });
   });
+  // TODO is this possible?
```
[the remark this answers is not in the document]
Comment (ikerreyes) [reply src/Components/AffectedClustersTable/AffectedClustersTable.spec.ct.js]: I have improved the TODO as it is not possible. However, it requires updating the data for the test, so I will do it in a separate PR. Is that OK?

**Pull request merged**